openslides_backend/action/actions/committee/create.py

```diff
@@ -22,5 +30,29 @@ class CommitteeCreate(CommitteeCommonCreateUpdateMixin, CreateAction):
             "forward_to_committee_ids",
             "receive_forwardings_from_committee_ids",
         ],
+        additional_optional_fields={"manager_ids": id_list_schema},
     )
     permission = OrganizationManagementLevel.CAN_MANAGE_ORGANIZATION
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        instance = super().update_instance(instance)
+        if "manager_ids" in instance:
+            manager_ids = instance.pop("manager_ids")
+            action_data: List[Dict[str, Any]] = []
+            for manager_id in manager_ids:
+                manager = self.datastore.get(
+                    FullQualifiedId(Collection("user"), manager_id), ["committee_ids"]
+                )
```

Comment:
A `get_many` would be better here, but maybe you want to change this together with https://github.com/OpenSlides/openslides-backend/pull/866#issuecomment-887277980

Reply:
Will later fix this.